dags/multipod/configs/gke_config.py

```diff
@@ -63,3 +64,67 @@ def get_gke_config(
       task_test_config=job_test_config,
       task_gcp_config=job_gcp_config,
   )
+
+
+def get_gke_maxtext_nightly_config(
+    tpu_version: TpuVersion,
+    tpu_cores: int,
+    tpu_zone: str,
+    time_out_in_min: int,
+    test_name: str,
+    docker_image: str,
+    test_owner: str,
+    cluster_name: str = ClusterName.V4_8_MULTISLICE_CLUSTER.value,
+    project_name: str = Project.TPU_PROD_ENV_MULTIPOD.value,
+    num_slices: int = 1,
+    dataset_name: metric_config.DatasetOption = metric_config.DatasetOption.XLML_DATASET,
+    dataset_project: str = Project.CLOUD_ML_AUTO_SOLUTIONS.value,
+    composer_project: str = Project.CLOUD_ML_AUTO_SOLUTIONS.value,
+) -> task.TpuXpkTask:
+  job_gcp_config = gcp_config.GCPConfig(
+      project_name=project_name,
+      zone=tpu_zone,
+      dataset_name=dataset_name,
+      dataset_project=dataset_project,
+      composer_project=composer_project,
+  )
+
+  current_time = datetime.datetime.now()
+  current_date = current_time.strftime("%Y-%m-%d")
+  current_datetime = current_time.strftime("%Y-%m-%d-%H-%M-%S")
+  base_output_directory = (
+      f"{gcs_bucket.XLML_OUTPUT_DIR}/maxtext/nightly/automated/{current_date}"
+  )
+  run_name = f"{num_slices}slice-V{tpu_version.value}_{tpu_cores}-maxtext-nightly-{current_datetime}"
```

Comment:
This will be DAG parsing time and not execution time, so it might be the same for all runs. Is that OK for now?

We will be improving the XPK GCS bucket handling so that it's truly unique for each test, but that's still WIP.

Reply:
This is fine for my usecase.

Reply:
I think this might actually be worse than being the same for all runs. What happens if the DAG is parsed again during a run? I believe this path will actually change between tasks when the DAG is rebuilt.

If it's fine to use the same directory for all runs, I suggest you just hardcode the path until @jonb377  improves XPK's handling of dynamically generated paths.

Reply:
I actually prefer that the `base_output_directory` name are different, we have an automated scripts to retrieve all xplane results within the same date.

https://screenshot.googleplex.com/BcNrN89EtKc4oro